changelog.json

```
@@ -1,5 +1,22 @@
 {
   "releases": {
+    "1.6.0": [
+      "[New] Help users add their first repo during onboarding - #6474",
+      "[New] \"No local changes\" view makes it easy to find and accomplish common actions - #6445",
+      "[Added] Include ability for users to add new repositories when there are none available - #5566 #6474",
```

Tentative feels on this line

```suggestion
      "[Improved] Repository list when no repositories found - #5566 #6474",
```